#in_file: "/home/oskar/navigation/data/left/ice_motion.bag" #Good baseline for left system
in_file: "/home/oskar/navigation/data/right/motion_1.bag" #Good baseline for left system

out_file: "/home/oskar/navigation/src/gtsam_nav/data/traj.txt"
max_time_interval: 300

gravity_norm: 9.831 # Norm in m/s^2, sign and reference frame is sorted in imu.cpp
initial_heading: 0 # Rads

# Uncertanties on initial values
initial_pos_sigma: [1, 1, 1] # These should be same as virtual height and gnss meas sigmas
initial_euler_sigma: [200, 200, 200] # Some big value, this should not affect much
initial_acc_bias_sigma: 10
initial_gyro_bias_sigma: 10 # TODO: Why is it trying to force us to huge bias?

# IMU noise parameters
imu_accel_noise_sigma: 1.0e-3         # Accelerometer noise (continous)
imu_gyro_noise_sigma: 1.0e-4          # Gyro noise (continous)
imu_accel_bias_rw_sigma: 1.0e-4       # No bias drift
imu_gyro_bias_rw_sigma: 1.0e-5        # No bias drift

imu_acc_scale: 1 # We should for sure calibrate the scale values at some point

# GNSS stuff
gnss_meas_sigma: 1                 # TODO: Sigma of measurement error

gnss_crs_source: "EPSG:4326"
gnss_crs_target: "EPSG:6052"

# Virtual height
virtual_height_interval: 1
virtual_height_sigma: 1